core/templates/pages/exploration-editor-page/modal-templates/exploration-metadata-modal.component.html

```diff
@@ -58,12 +58,14 @@ <h3>Add Some Details</h3>
         </mat-chip>
         <input placeholder="Add a new tag (using lowercase letters and spaces)..."
                pattern="[a-z]"
+               [maxLength]="30"
                class="e2e-test-chip-list-tags"
                [matChipInputFor]="chipList"
                [matChipInputSeparatorKeyCodes]="separatorKeysCodes"
                [matChipInputAddOnBlur]="addOnBlur"
                (matChipInputTokenEnd)="add($event)">
       </mat-chip-list>
+      <mat-hint *ngIf="explorationTags.length === 10">Can only contain 10 tags max</mat-hint>
```

Comment:
Can we make this `>= 10` just in case? And can we show the text with the red error CSS that is normally used?

Reply:
done

Reply:
![Screenshot from 2022-10-25 00-21-56](https://user-images.githubusercontent.com/77006897/197603167-48e72456-02a1-4d08-b6c7-ce7ae2d19bd1.png)